DECLARING_DEPENDENCIES.md

```diff
@@ -12,76 +12,29 @@ These recommendations apply to the following libraries:
 - [guava](https://github.com/google/guava)
 - [protobuf](https://github.com/protocolbuffers/protobuf)
 - [grpc-java](https://github.com/grpc/grpc-java)
-- [gax-java](https://github.com/googleapis/gax-java)
 - [google-http-java-client](https://github.com/googleapis/google-http-java-client)
-- [google-oauth-java-client](https://github.com/googleapis/google-oauth-java-client)
-- [google-api-java-client](https://github.com/googleapis/google-api-java-client)
 - [google-cloud-java](https://github.com/googleapis/google-cloud-java)
-- [beam](https://github.com/apache/beam)
-
-## Ensuring Compatibility
 
 To ensure that your own project uses compatible versions of these
-libraries, follow the guidance below for your build system.
-
-### Maven
-
-#### Use the requireUpperBoundDeps enforcer rule
```

Comment:
I don't think this should be removed. Users still need to use this enforcer rule for safety.